src/presentation/test/integration/localisation.test.ts

```diff
@@ -9,7 +9,7 @@ import { START_URL } from "../../../presentation/controller/global/Routing";
 
 describe("Localisation tests", () => {
   beforeEach(() => {
-    jest.resetAllMocks();
+    jest.clearAllMocks();
```

Comment:
when looking at the localisation tests, we had some previous problems with mocking when using the resetAllMocks (if we changed the way the mocking was done), clearAllMocks was the correct level to use, so just updated this while I remembered.